### PR TITLE
executor: 'INSERT IGNORE' should only ignore KeyExists error.

### PR DESCRIPTION
**Pull request by disksing**
When use 'INSERT IGNORE' to import data, all errors are ignored. It may cause data lost.

cc @coocood @shenli

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

File: kv/fault_injection.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+	t.cfg.RLock()
+	defer t.cfg.RUnlock()
+	if t.cfg.getError != nil {
+		return nil, t.cfg.getError
+	}
+	return t.Transaction.Get(k)
+}
+
+// InjectedSnapshot wraps a Snapshot with injections.
+type InjectedSnapshot struct {
```
Comment (coocood) [inline kv/fault_injection.go]: This is not used.

**Comment by disksing** [comment]
PTAL @coocood

**Comment by coocood** [comment]
LGTM

**Pull request merged**